src/config.py

```diff
@@ -0,0 +1,41 @@
+import os, yaml, pathlib
+
+BASE_DIR = pathlib.Path(__file__).resolve().parent.parent
+
+TOOLS_CFG_PATH = os.path.join(BASE_DIR, 'config', 'tools')
+TOOLS = {}
+for f in os.listdir(TOOLS_CFG_PATH):
+    tool_name,ext = os.path.splitext(f)
+    tool_cfg = os.path.join(TOOLS_CFG_PATH, f)
+    if os.path.isfile(tool_cfg) and ext == ".yaml":
+        with open(tool_cfg, 'r', encoding='utf-8') as ymlfile:
+            try:
+                TOOLS[tool_name] = yaml.safe_load(ymlfile)
+            except yaml.YAMLError as exc:
+                print(exc)
+TOOL_CHOICES = TOOLS.keys()
+
+DATASETS_CFG_PATH = os.path.join(BASE_DIR, 'config', 'datasets.yaml')
+DATASETS_PARENT = BASE_DIR
+with open(DATASETS_CFG_PATH, 'r') as ymlfile:
+    try:
+        DATASETS = yaml.safe_load(ymlfile)
+    except yaml.YAMLError as exc:
+        print(exc)
+DATASET_CHOICES = []
+for name,location in DATASETS.items():
+    if isinstance(location, dict) and 'subsets' in location:
+        for sname in location['subsets']:
+            DATASET_CHOICES.append(f"{name}/{sname}")
+    else:
+        DATASET_CHOICES.append(name)
+
+def is_remote_info(info):
+    return isinstance(info,dict) and 'url' in info and 'local_dir' in info
+
+def get_remote_info(info):
+    return (info['url'], info['local_dir'], info['subsets'])
+
+import pandas
+VULNERABILITY_MAP_PATH = os.path.join(BASE_DIR, 'src', 'output_parser', 'sarif_vulnerability_mapping.csv')
+VULNERABILITY_MAP = pandas.read_csv(VULNERABILITY_MAP_PATH)
```

Comment:
pandas dep should not be imported?

Reply:
When this is solved I think it can be merged

Reply:
Using pandas to read a csv file is like cracking a nut with a sledge-hammer. But this is the way how Smartbugs does it so far, not my idea, see https://github.com/smartbugs/smartbugs/blob/d6f70ad71610d2af27a9439d1fb8166c75ed548e/src/output_parser/SarifHolder.py .
My first impulse was to replace it, but then postponed it as it is a code change functionally not necessary. But it hurts, I agree, and will replace it.

Reply:
I meant that `import pandas` was missing but I did not see that it was in line 39. Sorry.

I don't think we need to replace it right now. Lets focus on something that has more value for the project.